kong/plugins/zipkin/opentracing.lua

```diff
@@ -58,7 +58,7 @@ function OpenTracingHandler:initialise_request(conf, ctx)
 			["http.url"] = ngx.var.scheme .. "://" .. ngx.var.host .. ":" .. ngx.var.server_port .. ngx.var.request_uri;
 			[ip_tag(ngx.var.remote_addr)] = ngx.var.remote_addr;
 			["peer.ipv6"] = nil;
-			["peer.port"] = tonumber(ngx.var.remote_port, 10);
+			["peer.port"] = ngx.var.remote_port;
```

Comment:
According to the [opentracing spec](https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table) this should be an integer.